### PR TITLE
SVG output now supports width and height attibute on top-level SVG ele…

### PR DESCRIPTION
**Pull request by fractalqb**
Addresses [#152](https://github.com/llgcode/draw2d/issues/152)

### CODE REVIEW

**Comment by llgcode** [comment]
Hi @fractalqb,
Thanks to submit this pull request see my comment in the code.

**Comment by fractalqb** [comment]
I had a look at the [commits](https://github.com/llgcode/draw2d/pull/153/commits/45f8f18303dbb4f9492ebaf409232d3fd89f24c2) but I couldn't find any comments. As I said, I'm not (yet) a github Pro!

File: draw2dsvg/svg.go [hunk 1]
```diff
@@ -35,6 +35,8 @@ const (
 type Svg struct {
 	XMLName  xml.Name `xml:"svg"`
 	Xmlns    string   `xml:"xmlns,attr"`
+	Width    float64  `xml:"width,attr,omitempty"`
```
[the remark this answers is not in the document]
Comment (fractalqb) [reply draw2dsvg/svg.go]: You are completely right and I was only going for some quick results. I changed it in my fork. Hope it's OK for a pull now.

**Comment by llgcode** [comment]
It looks ok for me, thanks @fractalqb

**Pull request merged**